snorkel/labeling/model/label_model.py

```diff
@@ -416,7 +460,7 @@ def score(
         -------
         >>> L = np.array([[1, 1, -1], [0, 0, -1], [1, 1, -1]])
         >>> label_model = LabelModel(verbose=False)
-        >>> label_model.train_model(L)
+        >>> label_model.fit(L, seed=123)
```

Comment:
is the seed necessary?

Reply:
Not anymore. The original lr (0.01) and n_epochs (100) were restored, and we seem to get consistent convergence for these examples.